remove even more high speed mmc stuff to get it working stable

diff --git a/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi b/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi
index 44a0346133cd..06a99953b804 100644
@@ -381,10 +381,10 @@
 	pinctrl-0 = <&mmc1_pins_default>;
 	pinctrl-1 = <&mmc1_pins_uhs>;
 	bus-width = <4>;
-	max-frequency = <200000000>;
+	max-frequency = <50000000>;
 	cap-sd-highspeed;
-	sd-uhs-sdr50;
-	sd-uhs-sdr104;
+//	sd-uhs-sdr50;
+//	sd-uhs-sdr104;
 	cd-gpios = <&pio 1 GPIO_ACTIVE_LOW>;
 	vmmc-supply = <&mt6397_vmch_reg>;
 	vqmmc-supply = <&mt6397_vmc_reg>;
 
